# HG changeset patch
# User Rik <rik@octave.org>
# Date 1472835307 25200
#      Fri Sep 02 09:55:07 2016 -0700
# Node ID 475b4cd1b789904e4ef2a4b8cf0b62a40bfc445f
# Parent  18b863e3b1e271b2ad2b03d4f9a10a33343442c6
doc: Correct docstring for '.**' operator (bug #48973)

* op-kw-docs: Change docstring for element-by-element power
operator to be '.**', not '.*' which is multiplication.

diff --git a/libinterp/op-kw-docs b/libinterp/op-kw-docs
--- a/libinterp/op-kw-docs
+++ b/libinterp/op-kw-docs
@@ -11,18 +11,16 @@
 ## WITHOUT ANY WARRANTY; without even the implied warranty of
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
-## FIXME: the information in this file is currently duplicated in help.cc.
-
 !
 @c libinterp/parse-tree/oct-parse.in.yy
 -*- texinfo -*-
 @deftypefn {} {} !
 Logical 'not' operator.
 @seealso{~, not}
 @end deftypefn
 ~
@@ -220,17 +218,17 @@ transpose, @emph{not} the complex conjug
 -*- texinfo -*-
 @deftypefn {} {} .*
 Element by element multiplication operator.
 @seealso{*, times}
 @end deftypefn
 .**
 @c libinterp/parse-tree/oct-parse.in.yy
 -*- texinfo -*-
-@deftypefn {} {} .*
+@deftypefn {} {} .**
 Element by element power operator.  If several complex results are possible,
 returns the one with smallest non-negative argument (angle).  Use
 @code{realpow}, @code{realsqrt}, @code{cbrt}, or @code{nthroot} if a
 real result is preferred.
 @seealso{**, ^, .^, power, realpow, realsqrt, cbrt, nthroot}
 @end deftypefn
 .^
 @c libinterp/parse-tree/oct-parse.in.yy
